shared/constants/network.js

```diff
@@ -70,6 +70,12 @@ export const TEST_ETH_TOKEN_IMAGE_URL = './images/black-eth-logo.svg';
 export const BNB_TOKEN_IMAGE_URL = './images/bnb.png';
 export const MATIC_TOKEN_IMAGE_URL = './images/matic-token.png';
 export const AVAX_TOKEN_IMAGE_URL = './images/avax-token.png';
+export const AETH_TOKEN_IMAGE_URL = './images/arbitrum.svg';
+export const FTM_TOKEN_IMAGE_URL = './images/fantom-opera.svg';
+export const HARMONY_ONE_TOKEN_IMAGE_URL = './images/harmony-one.svg';
+export const OPTIMISM_TOKEN_IMAGE_URL = './images/optimism.svg';
+export const PALM_TOKEN_IMAGE_URL = './images/palm.svg';
+export const BSC_TOKEN_IMAGE_URL = './images/bsc-filled.svg';
```

Comment:
We now have 2 images for BSC. See line 70. We should consolidate them and remove one

Reply:
Fixed: f0c4ca1d42e706eb710e1c2d54578a354d2de0ee